distributeaid/aggregated-public-information
# Script to Upload SDR Shipments Sheet

## What changed?

Added a new script to manage the upload.

Resolves #89 and #90 

## How can you test this?

Add the attached [shipments.csv](https://github.com/user-attachments/files/16644552/shipments.csv) export to the `src/scripts/import-sdr-shipments-sheet` folder, setup the `src/scripts/.env` file, then run the dev server in one terminal and `yarn script:import-sdr-shipments-sheet` in another.  The logs at the end should show you that most of the shipments were created (or already exists), and it's ok if there are a few data error cases.  If you see a lot of "Other" or "Error" results.
